plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -1,108 +1,19 @@
 /**
  * External dependencies
  */
-import { useMemo } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
-import { PaymentGateway } from '@woocommerce/data';
 
 /**
  * Internal dependencies
  */
 import './settings-payments-main.scss';
-import { PaymentMethod } from './components/payment-method';
-import { OtherPaymentMethods } from './components/other-payment-methods';
-import { PaymentsBannerWrapper } from '~/payments/payment-settings-banner';
+import { MainPaymentMethods } from '~/settings-payments/components/main-payment-methods';
+import { OtherPaymentMethods } from '~/settings-payments/components/other-payment-methods';
 
 export const SettingsPaymentsMain: React.FC = () => {
-	const [ paymentGateways, error ] = useMemo( () => {
-		const script = document.getElementById(
-			'experimental_wc_settings_payments_gateways'
-		);
-
-		try {
-			if ( script && script.textContent ) {
-				return [
-					JSON.parse( script.textContent ) as PaymentGateway[],
-					null,
-				];
-			}
-			throw new Error( 'Could not find payment gateways data' );
-		} catch ( e ) {
-			return [ [], e as Error ];
-		}
-	}, [] );
-
-	if ( error ) {
-		// This is a temporary error message to be replaced by error boundary.
-		return (
-			<div>
-				<h1>
-					{ __( 'Error loading payment gateways', 'woocommerce' ) }
-				</h1>
-				<p>{ error.message }</p>
-			</div>
-		);
-	}
-
 	return (
 		<div className="settings-payments-main__container">
-			<div id="wc_payments_settings_slotfill">
-				<PaymentsBannerWrapper />
-			</div>
-			<table className="form-table">
```

Comment:
I decided to extract it to `main-payment-methods.tsx` since we could have a lot of code here